modules/core/src/lib/attribute-manager.js

```diff
@@ -218,7 +218,7 @@ export default class AttributeManager {
 
     logFunctions.onUpdateStart({level: LOG_START_END_PRIORITY, id: this.id, numInstances});
     if (this.stats) {
-      this.stats.timeStart('attribute updates', this.id);
+      this.stats.get('DeckGL: Update Attributes').timeStart();
```

Comment:
Idea: Maybe the prefix `DeckGL: ` could be set when the `Stats` object is instantiated, so it wouldn't have to be repeated? The `Log` class from probe.gl works that way...

Reply:
What I was hoping to do eventually was have a single stats object passed from Luma to deck to track everything. The namespacing would prevent collisions in that case. But would we maybe prefer to keep them separate?

Reply:
Yes, I thought it would be nice to have them separate. Multiple libraries can each provide their own, and in addition the app can have its own, and the StatsWidget can handle all of them by just accepting a list of them...

Reply:
Sounds good. I'll remove the prefixes.